ios/FluentUI/Controls/ScrollView.swift

```diff
@@ -12,12 +12,12 @@ protocol ScrollableContainerView: AnyObject {
 }
 
 @available(*, deprecated, renamed: "ScrollView")
-public typealias MSScrollView = ScrollView
+public typealias MSScrollView = MSFScrollView
 
 /// `UIScrollView` subclass that automatically adjusts content insets when keyboard is shown/hidden to make sure scrollable area is always visible. Also provides methods to scroll any subview (or first responder if it's a subview) to visible area.
 @available(iOSApplicationExtension, unavailable)
 @objc(MSFScrollView)
-open class ScrollView: UIScrollView, ScrollableContainerView {
+open class MSFScrollView: UIScrollView, ScrollableContainerView {
```

Comment:
`@objc open class MSFScrollView: UIScrollView, ScrollableContainerView {`

Use just the **@objc** in front of the declaration since the name of the class and the objC are the same?